bot/university/university.py

```diff
@@ -27,6 +28,119 @@ def __init__(self, bot):
         self.bot = bot
         self._db_connector = DatabaseConnector(constants.DB_FILE_PATH, constants.DB_INIT_SCRIPT)
 
+    @commands.group(name='module', invoke_without_command=True)
+    @command_log
+    async def toggle_module(self, ctx: commands.Context, *, str_modules: str):
+        """Command Handler for the `module` command.
+
+        Allows members to assign/remove so called mod roles to/from themselves. This way users can toggle text channels
+        about specific courses to be visible or not to them. When the operation is finished, SAM will send an overview
+        about the changes he did per direct message to the user who invoked the command.
+        If the command is invoked outside of the configured role channel, the bot will post a short info that this
+        command should only be invoked there and delete this message shortly after.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            str_modules (str): A string containing abbreviations of all the modules a user would like to toggle.
+        """
+        if ctx.channel.id != constants.CHANNEL_ID_ROLES:
+            ctx.channel.send(value=f"Dieser Befehl wird nur in <#{constants.CHANNEL_ID_ROLES}> unterstützt. Bitte "
```

Comment:
Maybe DM the user here, so other users are not confused if they do other commands in the channel, where this was posted